src/components/Navbar/Navbar.css

```diff
@@ -0,0 +1,83 @@
+.top-bar
+{
+    z-index: 15;
```

Comment:
suggestion: you can crate a constant for z-indexes and use them.

Reply:
Done, thanks for suggesting this, will make z-indexes more manageable